timemachine/fe/system.py

```diff
@@ -183,8 +183,8 @@ def get_U_fns(self):
             self.bond,
             self.angle,
             self.torsion,
-            self.chiral_atom,
-            self.chiral_bond,
+            # self.chiral_atom,
```

Comment:
# intentionally disabled until we fix MCS bits (and enforce chirality checking)

Reply:
Could we add this comment to the code?

Reply:
willfix